Firmware/planner.cpp

```diff
@@ -714,10 +714,7 @@ void plan_buffer_line(float x, float y, float z, const float &e, float feed_rate
   // Rest here until there is room in the buffer.
   if (block_buffer_tail == next_buffer_head) {
       do {
-          manage_heater(); 
-          // Vojtech: Don't disable motors inside the planner!
-          manage_inactivity(false); 
```

Comment:
beware - manage_inactivity(**false**); at this spot

Reply:
I suspect this is a bug. It doesnt make sense to allow the motors to be disabled here.